app/styles/app.scss

```diff
@@ -736,79 +736,77 @@ hr {
     }
 }
 
-#preprint-form-subjects {
-    .subject {
-        background-color: $brand-primary;
-        display: inline-block;
-        color: white;
-        overflow: hidden;
-        padding: 0 2rem;
-        margin-right: 2rem;
-        margin-bottom: 1rem;
-        border-radius: 2px;
-        transition: background-color 0.1s linear;
-        &:hover {
+.right-arrow {
+    border-top: 14px solid transparent;
+    border-bottom: 15px solid transparent;
+    border-left: 15px solid white;
+    position: absolute;
+    display: inline;
+    &:before {
+        content: '';
+        border-top: 14px solid transparent;
+        border-bottom: 15px solid transparent;
+        border-left: 15px solid $brand-primary;
+        position: absolute;
+        margin-left: -16px;
+        margin-top: -14px;
```

Comment:
Any reason for the slight variation in border and margin sizes here? (14, 15, 16 px)? Asking in case this relates to the slightly alignment issues I was seeing.

Also, is there any particular reason for the move from `rem` to `px` units? (eg were we deliberately trying to prevent rescaling in this case?)